src/ai/backend/manager/cli/image.py

```diff
@@ -59,10 +59,11 @@ def forget(cli_ctx, canonical_or_alias, architecture) -> None:
 @cli.command()
 @click.argument("canonical_or_alias")
 @click.argument("architecture")
+@click.option("--remove-from-registry", is_flag=True, help="Remove the image from the registry.")
 @click.pass_obj
-def purge(cli_ctx, canonical_or_alias, architecture) -> None:
+def purge(cli_ctx, canonical_or_alias, architecture, remove_from_registry) -> None:
```

Comment:
Can't you add a type in a function of click?

Reply:
It’s simply because the existing code lacked type hints. Let’s add type hints.